From eab42a8a788c70b4fb3409602c906e88a39e9c02 Mon Sep 17 00:00:00 2001
From: "mason.zou" <mason.zou@sophgo.com>
Date: Wed, 6 May 2026 19:49:25 +0800
Subject: [PATCH] [feat](teaisp): sync teaisp bnr code [description ]:NA [root 
  cause]:NA [JIRA      ID]:NA [chip project]:cv184x [side effects]:NA
 [Affected *.ko/*.so/*.a] :NA
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change-Id: Ieca9b60f57d173556fa00356c5aa1de6d2029a17
Reviewed-on: https://gerrit-ai.sophgo.vip:8443/162645
Reviewed-by: 振兴 叶 <zhenxing.ye@sophgo.com>
---
 cmake/middleware.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/middleware.cmake b/cmake/middleware.cmake
index 11798e96..5894ec32 100644
--- a/cmake/middleware.cmake
+++ b/cmake/middleware.cmake
@@ -209,6 +209,7 @@ elseif(${CVI_PLATFORM} STREQUAL "CV184X")
         ${MIDDLEWARE_SDK_ROOT}/lib/libvpss.so
         ${MIDDLEWARE_SDK_ROOT}/lib/libvo.so
         ${MIDDLEWARE_SDK_ROOT}/lib/libisp.so
+        ${MIDDLEWARE_SDK_ROOT}/lib/libisp_algo.so
         ${MIDDLEWARE_SDK_ROOT}/lib/libvdec.so
         ${MIDDLEWARE_SDK_ROOT}/lib/libvenc.so
         ${MIDDLEWARE_SDK_ROOT}/lib/libawb.so
@@ -226,6 +227,7 @@ elseif(${CVI_PLATFORM} STREQUAL "CV184X")
         ${MIDDLEWARE_SDK_ROOT}/lib/libvpss.a
         ${MIDDLEWARE_SDK_ROOT}/lib/libvo.a
         ${MIDDLEWARE_SDK_ROOT}/lib/libisp.a
+        ${MIDDLEWARE_SDK_ROOT}/lib/libisp_algo.a
         ${MIDDLEWARE_SDK_ROOT}/lib/libvdec.a
         ${MIDDLEWARE_SDK_ROOT}/lib/libvenc.a
         ${MIDDLEWARE_SDK_ROOT}/lib/libawb.a
@@ -239,12 +241,10 @@ elseif(${CVI_PLATFORM} STREQUAL "CV184X")
     if(${CONFIG_DUAL_OS} STREQUAL "OFF")
         set(MIDDLEWARE_LIBS
           ${MIDDLEWARE_LIBS}
-          ${MIDDLEWARE_SDK_ROOT}/lib/libisp_algo.so
           ${MIDDLEWARE_SDK_ROOT}/lib/libsns_full.so
           ${MIDDLEWARE_SDK_ROOT}/lib/libsensor_i2c.so)
         set(MIDDLEWARE_LIBS_STATIC
           ${MIDDLEWARE_LIBS_STATIC}
-          ${MIDDLEWARE_SDK_ROOT}/lib/libisp_algo.a
           ${MIDDLEWARE_SDK_ROOT}/lib/libsns_full.a
           ${MIDDLEWARE_SDK_ROOT}/lib/libsensor_i2c.a)
     endif()
-- 
2.25.1

